### PR TITLE
added another religon

### PR DESCRIPTION
**Pull request by nikk735**
## About The Pull Request

adds Taoism, adds a slice of gondola meat into your bag if you choose Taoism 
also removed anti gondola propaganda

## Why It's Good For The Game

we just needed to have a gondola religion its completely necessary for game play 

## Changelog
:cl:
add gondola religon
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/chaplain.dm [hunk 1]
```diff
@@ -99,6 +99,11 @@
 			B.name = pick("Toolbox Manifesto","iGlove Assistants")
 		if("weeaboo","kawaii")
 			B.name = pick("Fanfiction Compendium","Japanese for Dummies","The Manganomicon","Establishing Your O.T.P")
+		if("gondola","taoism")
+			B.name = "Taoism for Dummies"
+			B.deity_name = pick("Gondola")
```
Comment (St0rmC4st3r) [inline code/modules/jobs/job_types/chaplain.dm:104]: You disgust me.

Comment (St0rmC4st3r) [inline code/modules/jobs/job_types/chaplain.dm]:
```suggestion
			B.deity_name = "Gondola"
```

**Review by St0rmC4st3r — CHANGES_REQUESTED** [review]

**Comment by zeskorion** [comment]
Looks like you just want a way to get roundstart Tranquility

**Comment by St0rmC4st3r** [comment]
> Looks like you just want a way to get roundstart Tranquility

What if it also gave a ton of krokodil and pax roundstart?

**Comment by nikk735** [comment]
zesk you are exactly right

**Comment by nikk735** [comment]
i copy paste didnt mean to leave that in

**Comment by nikk735** [comment]
there  needs to be a non admin way to start the gondola revolution

**Comment by zeskorion** [comment]
> there needs to be a non admin way to start the gondola revolution

just go to space

**Comment by nikk735** [comment]
> > there needs to be a non admin way to start the gondola revolution
> 
> just go to space

how can a chaplain trying to spread peace and tao go to space?

**Comment by nikk735** [comment]
the game is unplayable if you cant eat gondola meat within the first 3 seconds of the round start

**Comment by zeskorion** [comment]
> the game is unplayable if you cant eat gondola meat within the first 3 seconds of the round start

then make the religion add the reagent to you, not give you gondola steak

**Comment by nikk735** [comment]
Well you can’t convert others to the gondola religion then

**Comment by nikk735** [comment]
this is kinda a superior idea tho

**Comment by ike709** [comment]
Lame PR, plus you can sometimes find a gondola in random maint.

**Pull request closed without merging**